pkg/apis/core/validation/validation.go

```diff
@@ -2632,6 +2632,19 @@ const (
 	MaxDNSSearchListChars = 256
 )
 
+func validateReadinessGates(readinessGates []core.PodReadinessGate, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if !utilfeature.DefaultFeatureGate.Enabled(features.PodReadinessGates) {
+		return append(allErrs, field.Forbidden(fldPath, "ReadinessGates: Pod Ready++ is disabled by feature gate"))
```

Comment:
nit: Prefer describing this feature with `PodReadinessGates` rather than `Pod Ready++`.

Reply:
Acked. Fixed in the dependency PR https://github.com/kubernetes/kubernetes/pull/64057